encodings/fsst/js/src/fsstDecoder.ts

```diff
@@ -0,0 +1,23 @@
+export class FsstDecoder {
```

Comment:
Let's simplify and get rid of that class and just export the function like `decodeFsst`